config/samples/falcon_v1alpha1_falconnodesensor.yaml

```diff
@@ -14,5 +14,5 @@ spec:
     client_id: PLEASE_FILL_IN
     client_secret: PLEASE_FILL_IN
     cloud_region: autodiscover
-  node: {}
-  falcon: {}
+  node: {} # required even if empty
+  falcon: {} # required even if empty
```

Comment:
What about dropping this requirement instead of documenting it?

Reply:
I'm cool with that if you and @redhatrises are. Let me know what you decide and I'll go from there.

Reply:
Make sense... especially as we are setting the defaults types so that only a minimum configurations need to be set.